pkg/cmd/server/bootstrappolicy/controller_policy.go

```diff
@@ -16,6 +16,9 @@ var (
 	controllerRoles = []rbac.ClusterRole{}
 	// controllerRoleBindings is a slice of roles used for controllers
 	controllerRoleBindings = []rbac.ClusterRoleBinding{}
+
+	ReadWrite = []string{"get", "list", "watch", "create", "update", "patch", "delete", "deletecollection"}
```

Comment:
We need the same for auditing. So we probably make them available in `k8s.io/apiserver/pkg/endpoints/request`